sensors/movementsensor.go

```diff
@@ -24,7 +24,7 @@ const (
 )
 
 var (
-	defaultTime = time.Time{}
+	undefinedTime = time.Time{}
```

Comment:
Renamed this since I kept getting confused with `defaultTime`, assuming it was set to a specific time, when really it's just used to denote that it was not set to any meaningful value yet.